src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -263,6 +264,11 @@
 					.examples("on tool break:")
 					.since("2.1.1");
 		}
+		Skript.registerEvent("Item Damage", SimpleEvent.class, PlayerItemDamageEvent.class, "[player] tool damag(e|ing)", "[player] damag(e|ing) (a|the|) tool")
```

Comment:
why no syntax for "item damage", also should use `[ ]` instead of empty option

Reply:
> item damage

good call

> empty option

My bad. I just copied the line from the event above to match it. Will change it up quickly now.